src/flow-manager.c

```diff
@@ -422,7 +422,7 @@ static uint32_t FlowTimeoutHash(FlowManagerTimeoutThread *td, SCTime_t ts, const
 #define TYPE uint32_t
 #endif
 
-    const uint32_t ts_secs = SCTIME_SECS(ts);
+    const time_t ts_secs = SCTIME_SECS(ts);
```

Comment:
the second tracking in the flow engine is currently all u32, so I would suggest leaving this as u32 until we want to move it to a larger type

Reply:
ok